src/Services/SwaggerService.php

```diff
@@ -281,6 +281,53 @@ protected function markAsDeprecated(array $annotations)
         $this->item['deprecated'] = Arr::get($annotations, 'deprecated', false);
     }
 
+    protected function saveResponseSchema(?array $content, int $code)
+    {
+        if (empty($content)) {
+            return;
+        }
+
+        $schemaProperties = [];
+        $action = Str::ucfirst($this->getActionName($this->uri));
+        $schemaType = 'object';
+
+        if (array_is_list($content)) {
```

Comment:
let's move it to the separate methods

```suggestion
        if (array_is_list($content)) {
            $schemaProperties = $this->saveListReponseDifinitions();
        } else {
            $schemaProperties = $this->saveObjectResponseDifinitions();
        }
```